### ISSUE TITLE
Hardcoded ROOTFS_SECTOR in mksdcardimg.sh

### ISSUE THREAD

**Issue by frep** [radxa/rock-bsp#2]
I really like the rock-bsp repository. So, I would like to help improve it. (In fact, it's way better than my code to generate images, so I plan to change my building-process using a branch of this repository)

While having a look at the `mksdcarimg.sh` I noticed, that the `ROOTFS_SECTOR` is hardcoded in the script. I think this is something between bad and wrong, since the `ROOTFS_SECTOR` depends on the partition informations of linuxroot located in the `parameter` file. 

In my opinion, the value should be read out of the `parameter` file (and then added with the loader offset of 0x2000).

**Comment by hipboi** [comment]
@freq, right, we need a script to parse the parameter to get the ROOTFS_SECTOR, but for now 32MB offset is enough for us.

**Comment by frep** [comment]
I start to see now, why it works the way it is. 

I made the wrong assumtion, that when the loader is looking for the memory location of the rootfs it checks the offset of linuxroot relative to its own location. But it's way simpler than that! The parameter file contains the information (which I overlooked until now) that the root is located at partition1 of the sdcard. So as long as we write the partition table accordingly (which is provided by the script already) we are fine. 
I think, I have learned something today! :-)

But even it is not so serious (meaning the hardcoded sector is more "bad" than "wrong") the point that the sector should be determined by the parameter file, stays valid. The issue is not solved, but has probably no priority at all. 

(No priority for me either, since rock2 should finally arrive TODAY! Can't wait to see kali running on it!)
